CHANGELOG.md

```diff
@@ -42,6 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - "key" sub-commands to "generate" and "restore" a master private key
 - "key" sub-command to "derive" an extended public key from a master private key
 - "repl" command now has an "exit" sub-command
+- `CompactFilterOpts` to `WalletOpts` to enable compact-filter blockchain configuration 
```

Comment:
This should be moved up under the "unreleased" section, since it will become part of the following release

Reply:
eh. I should have seen that. Updated in revised.